monai/transforms/post/array.py

```diff
@@ -858,3 +871,97 @@ def __call__(self, image: NdarrayOrTensor) -> torch.Tensor:
         grad = torch.cat([grad_h, grad_v], dim=1)
         grad, *_ = convert_to_dst_type(grad.squeeze(0), image_tensor)
         return grad
+
+
+class GetInstancelabelledSegMap(Transform):
+    """
+    Process Nuclei Prediction with XY Coordinate Map.
+
+    Args:
+        threshold_pred: threshold the float values of prediction to int 0 or 1 with specified theashold. Defaults to 0.5.
+        threshold_overall: threshold the float values of overall gradient map to int 0 or 1 with specified theashold.
+            Defaults to 0.4.
+        min_size: objects smaller than this size are removed. Defaults to 10.
+        sigma: std. could be a single value, or `spatial_dims` number of values. Defaults to 0.4.
+        kernel_size: the size of the Sobel kernel. Defaults to 17.
+        radius: the radius of the disk-shaped footprint. Defaults to 2.
+
+    """
+
+    backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
```

Comment:
I think the backend should be `[TransformBackends.TORCH]` as you only support torch computation here.

Thanks.